configs/sites/tier1/acorn/packages.yaml

```diff
@@ -41,3 +41,6 @@
       externals:
       - spec: grep@3.1
         prefix: /usr
+    esmf:
```

Comment:
Didn't we discuss making this change in configs/common/packages.yaml for all systems on develop after the release? If so, still ok to merge this and then replace in a follow-up PR.

Reply:
[That's already been changed](https://github.com/JCSDA/spack-stack/blob/8faa9494b3f6dc7f02f60832b43103b44a000e74/configs/common/packages.yaml#L74) as part of my last PR. I can remove this redundant require from the Acorn config later.

Reply:
Done in 061730c8358f677ef25db77bbce54b22dcaf4cb7, I had to push the submodule pointer update anyway